scalalib/src/ScalaModule.scala

```diff
@@ -104,6 +104,14 @@ trait ScalaModule extends JavaModule { outer =>
    */
   protected def mandatoryScalacOptions = T { Seq.empty[String] }
 
+  /**
+   * Scalac options to active the compiler plugins.
+   */
+  protected def enablePluginScalacOption: Target[Seq[String]] = T {
+    val resolvedJars = resolveDeps(scalacPluginIvyDeps.map(_.map(_.exclude("*" -> "*"))))()
+    resolvedJars.iterator.toSeq.map(jar => s"-Xplugin:${jar}")
```

Comment:
`jar` is a `PathRef` here. Also if you delay the creation of the Seq it avoid allocating an extra `Seq`
```suggestion
    resolvedJars.iterator.map(jar => s"-Xplugin:${jar.path}").toSeq
```

Reply:
Good catch.

Reply:
To my defense, I tried to use BSP in IJ... didn't work so well, obviously.